tigerpath/majors_and_certificates/majors/MOL.json

```diff
@@ -135,7 +137,7 @@
           "name": "Quantitative",
           "max_counted": 1,
           "min_needed": "ALL",
-          "explanation": "Students satisfy the quantitative requirement by taking one course in statistics (SML 201, NEU 314, or ORF 245) and one course in either computer science (COS 126 or above) or math (MAT 103, 104, 175, 192, or any 200-level MAT course). SML 201 and COS 126 are the recommended choices for most students. AP credit cannot be used toward the fulfillment of the quantitative requirement. Courses taken at other institutions can be substituted for the second required course (but not for the statistics course), if approved by the corresponding department.",
+          "explanation": "Students satisfy the quantitative requirement by taking one course in statistics (SML 201, ORF 245, or MOL 290) and one course in either computer science (COS 126 or above) or math (MAT 103, 104, 175, 192, or any 200-level MAT course). SML 201 and COS 126 are the recommended choices for most students. AP credit cannot be used toward the fulfillment of the quantitative requirement. Courses taken at other institutions can be substituted for the second required course (but not for the statistics course), if approved by the corresponding department.",

```

Comment:
https://molbio.princeton.edu/undergraduate/major/requirements

Quantitative  Students satisfy the quantitative requirement by taking one course in statistics (SML 201, ORF 245, or MOL 290) and one course in either computer science (COS 126 or above) or math (MAT 103, 104, 175, 192, or any 200-level MAT course). SML 201 and COS 126 are the recommended choices for most students. AP credit cannot be used toward the fulfillment of the quantitative requirement. Courses taken at other institutions can be substituted for the second required course (but not for the statistics course), if approved by the corresponding department.